src/solc_0.8/polygon/child/asset/PolygonAssetERC721.sol

```diff
@@ -118,15 +127,27 @@ contract PolygonAssetERC721 is BaseERC721, IPolygonAssetERC721 {
         address from,
         address to,
         uint256[] calldata ids
-    ) public override(BaseERC721, IERC721Base) {
+    ) public override(BaseERC721, IERC721Base) onlyAllowedOperator(from) {
         BaseERC721.batchTransferFrom(from, to, ids);
     }
 
     function approveFor(
         address from,
         address operator,
         uint256 id
-    ) public override(BaseERC721, IERC721Base) {
+    ) public override(BaseERC721, IERC721Base) onlyAllowedOperatorApproval(operator) {
         BaseERC721.approveFor(from, operator, id);
     }
+
+    function _msgSender() internal view override(BaseERC721, ContextUpgradeable) returns (address) {
```

Comment:
I'm not sure why this has been added? This should return `BaseERC721._msgSender()`